less/global.less

```
@@ -17,19 +17,34 @@ h1, h2, h3, h4, h5 {
   color: @header-font-color;
 }
 
+h1{
+  font-size: 3.7rem;
+}
+
 h2 {
   font-size: 3.2rem;
 }
 
 h3 {
-  font-size: 2.8rem;
+  font-size: 2.3rem;
+  padding: 15px;
```

```suggestion
  padding: 15px 1.5%;
```
Top and bottom is OK to hardcode, left right should be a %.